RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotFromDB.cc

```diff
@@ -0,0 +1,56 @@
+/**_________________________________________________________________
+   class:   OnlineBeamSpotFromDB.cc
+   package: RecoVertex/BeamSpotProducer
+   
+
+
+ author: Francisco Yumiceva, Fermilab (yumiceva@fnal.gov)
+
+
+________________________________________________________________**/
+
+// C++ standard
+#include <string>
+// CMS
+#include "DataFormats/BeamSpot/interface/BeamSpot.h"
+#include "RecoVertex/BeamSpotProducer/plugins/OnlineBeamSpotFromDB.h"
```

Comment:
This could be just
```suggestion
#include "OnlineBeamSpotFromDB.h"
```
or, event better, the contents of the header included in this file.